### PR TITLE
[WIP] Remove Swift 2.3 support 💣

### PR DESCRIPTION
**Pull request by marcelofabri**
👋

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,10 +3,8 @@ osx_image: xcode8.2
 
 before_script:
   - bundle install
-  - cd Bootstrap ; bundle exec pod install --repo-update ; cd ..
-  - cd BootstrapSwift3 ; bundle exec pod install ; cd ..
+  - cd BootstrapSwift3 ; bundle exec pod install --repo-update ; cd ..
```
Comment (ashfurrow) [inline .travis.yml]: Do we want to rename this directory while we're at it?

Comment (marcelofabri) [inline .travis.yml]: I thought about it, but I wasn't sure how git would handle it

**Comment by fsaragoca** [comment]
@marcelofabri there is a link for [DynamicSizeTests.swift](https://github.com/ashfurrow/Nimble-Snapshots/blob/master/Bootstrap/BootstrapTests/DynamicSizeTests.swift) in [README.md](https://github.com/marcelofabri/Nimble-Snapshots/blob/bye-swift2.3/README.md), you should probably remove it since you're deleting the `Bootstrap` folder, or I can remove it in #81.

**Comment by marcelofabri** [comment]
@fsaragoca I thought all the tests were implemented on both sample projects, I'll have to double check it before merging

**Pull request closed without merging**